src/components/app/DragAndDrop.css

```diff
@@ -0,0 +1,38 @@
+.dragAndDropArea {
```

Comment:
I think some of the CSS files are lacking the license preamble, but we should add that to this file.:

```
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```